Orange/widgets/visualize/owmosaic.py

```diff
@@ -512,6 +512,18 @@ def update_graph(self):
         spacing = self.SPACING
         bar_width = self.BAR_WIDTH
 
+        def get_counts(attr_vals, values):
+            if attr_vals == "":
+                counts = [conditionaldict[val] for val in values]
```

Comment:
While draw_data used conditionaldict[val], draw_text used conditionaldict.get(val, 1). Could you elaborate on why indexing is sufficient?